scripts/privateKeyToAddress.js

```diff
@@ -0,0 +1,12 @@
+const { privateKeyToAddress } = require('../src/utils/utils')
+
+const privateKey = process.argv[2]
```

Comment:
the private key should be get from the environment variable `VALIDATOR_ADDRESS_PRIVATE_KEY` as it was described in https://github.com/poanetwork/token-bridge/issues/107.

Reply:
Done.